CanHazFunny/CanHazFunny/Jester.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+
+namespace CanHazFunny;
+
+public class Jester
+{
+    private readonly IJokeOutput _JokeOutput;
+    private readonly IJokeService _JokeService;
+
+    public Jester(IJokeOutput output, IJokeService service)
```

Comment:
```suggestion
    public Jester(IJokeOutput jokeOutput, IJokeService jokeService)
```
Small nit, but I'd recommend keeping the names the same and just varying the casing. This is more conventional.